pjmedia/src/pjmedia/transport_srtp_dtls.c

```diff
@@ -1492,8 +1515,10 @@ static pj_status_t dtls_media_start( pjmedia_transport *tp,
 	    ap.user_data = ds->srtp;
 
 	    /* Attach ourselves to member transport for DTLS nego. */
-	    if (!pj_sockaddr_has_addr(&ds->rem_addr))
-		get_rem_addrs(ds, sdp_remote, media_index);
+            if (!pj_sockaddr_has_addr(&ds->rem_addr)) {
+                get_rem_addrs(ds, sdp_remote, media_index, &ds->rem_addr,
+                              &ds->rem_rtcp);
+            }
 
 	    if (pj_sockaddr_has_addr(&ds->rem_addr))
 		pj_sockaddr_cp(&ap.rem_addr, &ds->rem_addr);
```

Comment:
for consistency, shouldn't we also set `peer_use` (to `MANDATORY`) in `media_start()`?

Reply:
ok